main.py

```
@@ -117,19 +149,23 @@ async def get_new_notice(authorization: str = Header(None), notice_type: str = N
 @app.get("/scraper/categories")
 async def get_scraper_categories(authorization: str = Header(None)):
     validate_api_key(authorization)
     return await db_manager.read_category_list()
 
+
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run("main:app", host="0.0.0.0", port=8000, reload=True)
\ No newline at end of file
+
+    uvicorn.run("main:app", host="0.0.0.0", port=8000, reload=True)
```

Ruff (0.14.8)

Possible binding to all interfaces

(S104)